contracts/0.8.9/test_helpers/oracle/HashConsensusTimeTravellable.sol

```diff
@@ -72,4 +72,12 @@ contract HashConsensusTimeTravellable is HashConsensus {
     function advanceTimeByEpochs(uint256 numEpochs) external {
         _time += SECONDS_PER_SLOT * SLOTS_PER_EPOCH * numEpochs;
     }
+
+    function getFrameConfigFastLaneLengthSlots() external view returns (uint256 fastLaneLengthSlots) {
```

Comment:
this setting needs to be added as a return parameter to the `getFrameConfig` function of the main `HashConsensus` contract, I forgot to do this. could you please implement this as part of this PR?

Reply:
Added to contract